recipes-bsp/u-boot/u-boot_%.bbappend

```diff
@@ -0,0 +1,3 @@
+UBOOT_MACHINE = "rpi_arm64_config"
+
+COMPATIBLE_MACHINE:raspberrypi5 = "raspberrypi5"
```

Comment:
Do we really need 2 files bb and bbappend?

Reply:
Current meta-raspberrypi layer doen't support u-boot for rpi5 and disable use uboot for rpi5.
First is setup correct branch of u-boot. Second one allows use u-boot for rpi5 and setup u-boot configuration for rpi5

Reply:
Do you have any examples of the same approach from some of the Yocto layers?
When you create a recipe and alter it with bbappend in the same dir?
I'm sorry but this approach looks weird for me

Reply:
This is strange but it doesn't work if merge .bb and .bbapend files